app/src/main/kotlin/com/simplemobiletools/calculator/activities/CryptoActivity.kt

```diff
@@ -0,0 +1,149 @@
+import android.view.View
+import com.simplemobiletools.calculator.R
+import com.simplemobiletools.commons.extensions.copyToClipboard
+import com.simplemobiletools.commons.extensions.restartActivity
+import com.simplemobiletools.commons.extensions.value
+import com.simplemobiletools.commons.extensions.performHapticFeedback
+import com.simplemobiletools.commons.extensions.toast
+import com.simpletools.calculator.commons.activities.SimpleActivity
+import com.simpletools.calculator.commons.extensions.config
+import com.simpletools.calculator.commons.extensions.updateViewColors
+import com.simpletools.calculator.commons.helpers.Calculator
+import kotlinx.android.synthetic.main.activity_crypto.*
+import me.grantland.widget.AutofitHelper
+import android.widget.NumberPicker
+import com.simplemobiletools.calculator.helpers.CryptoCalculatorImpl
+import com.simpletools.calculator.commons.helpers.Formatter
+
+class CryptoActivity : SimpleActivity(), Calculator {
+    private var storedTextColor = 0
+    private var vibrateOnButtonPress = true
+    private var storedUseEnglish = false
+
+    private var cryptoFROM: String = ""
+    private var cryptoTO: String = ""
+
+    lateinit var calc: CryptoCalculatorImpl
+
+    @SuppressLint("MissingSuperCall")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_crypto)
+
+        calc = CryptoCalculatorImpl(this, applicationContext)
+        updateViewColors(crypto_holder, config.textColor)
+
+        getButtonIds().forEach {
+            it.setOnClickListener {
+                calc.numpadClicked(it.id); checkHaptic(it)
+            }
+        }
+
+        var numberPickerFROM: NumberPicker = numberPicker1
+        var numberPickerTO: NumberPicker = numberPicker2
+
+        // Values of each of the Number Pickers
+        val valuesFROM = arrayOf("Bitcoin", "Ethereum", "Ripple", "Litecoin", "USD")
+        val valuesTO = arrayOf("Ethereum", "Ripple", "Litecoin", "USD", "Bitcoin")
+
+        // Setting default value of each Number Picker
+        numberPickerFROM.minValue = 0
+        numberPickerTO.minValue = 0
+
+        // Setting size of each Number Picker
+        numberPickerFROM.maxValue = valuesFROM.size - 1
+        numberPickerTO.maxValue = valuesTO.size - 1
+
+        // Setting Number Pickers to the values
+        numberPickerFROM.displayedValues = valuesFROM
+        numberPickerTO.displayedValues = valuesTO
+
+        // Sets whether the selector wheel wraps when reaching the min/max value.
+        numberPickerFROM.wrapSelectorWheel = true
+        numberPickerTO.wrapSelectorWheel = true
+
+        val cryptoSymbolHashMap = HashMap<String, String>()
+        cryptoSymbolHashMap.put("Ethereum", "ETH")
+        cryptoSymbolHashMap.put("Bitcoin", "BTC")
+        cryptoSymbolHashMap.put("Ripple", "XRP")
+        cryptoSymbolHashMap.put("Litecoin", "LTC")
+        cryptoSymbolHashMap.put("USD", "USD")
```

Comment:
No bitconnect 🧐

Reply:
Unfortunately our application does not support scammers

Reply:
:'(